src/main/java/com/example/domainmodelopdrachtspringbootles11/controller/TelevisionController.java

```diff
@@ -0,0 +1,67 @@
+package com.example.domainmodelopdrachtspringbootles11.controller;
+
+import com.example.domainmodelopdrachtspringbootles11.exceptions.RecordNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+
+@RestController
+public class TelevisionController {
+    ArrayList<String >televisionDataBase = new ArrayList<>();
+
+    @GetMapping("television")
+    public ResponseEntity<ArrayList<String>> getAllTelevisions() {
+        televisionDataBase.add("Sony Bravia");
```

Comment:
Dit is de GET-mapping, we willen dus iets getten. Het is dan niet de bedoeling dat je hier iets in de database gaat zetten, maar juis returned wat er reeds in de database staat.